Neos.ContentRepository/Classes/Migration/Command/NodeCommandController.php

```diff
@@ -86,7 +86,7 @@ class NodeCommandController extends CommandController
      * @param boolean $confirmation Confirm application of this migration, only needed if the given migration contains any warnings.
      * @param string $direction The direction to work in, MigrationStatus::DIRECTION_UP or MigrationStatus::DIRECTION_DOWN
      * @return void
-     * @see neos.contentrepository.migration:node:migrationstatus
+     * @see neos.contentrepository.migration:node:migrate
```

Comment:
In the [KickstartCommandController](https://github.com/neos/flow-development-collection/blob/8.3/Neos.Kickstarter/Classes/Command/KickstartCommandController.php) its done the same way. I thought this would be a hint. So i know what to look for in the neos documentation on readthedocs.

Reply:
No, this is output with the help and in this case is intended to point to the `node:migrationstatus` command as a "counter-part" to the `node:migrate` command…